core/test/missingVariables.test.js

```diff
@@ -307,6 +307,55 @@ transport . avion . usager:
 		])
 		expect(result).to.have.lengthOf(2)
 	})
+
+	it("Parent's other descendands in sums should not be included as missing variables, even when parent evluation is triggered by a comparison", function () {
```

Comment:
On peut peut-être merger ça en ajoutant un `it.skip` et créer une issue pour régler le soucis?